app/db.py

```diff
@@ -63,38 +69,307 @@ def load_tweet_ids(self):
+                func.max(Conversations.conversation_status).label("status"),
+                Conversations.root_tweet_id
+            ).group_by(
+                Converstaions.root_tweet_id
+            ).cte('wow')
+
+            query2 = select(
+                Conversations
+            ).join(
+                query1,
+                query1.c.root_tweet_id == Conversations.root_tweet_id
+            ).filter(
+                Conversations.conversation_status == query1.c.status
+            )
+
+            data = session.execute(query2)
+
+        return [i[0] for i in data.fetchall()]
+
+
+    def update_conversation_checks(self, root_id):
+        """ iterates conversation_checks column of matching root_tweet_id """
+        query =(
+            update(Conversations).
+            where(Conversations.root_tweet_id == str(root_id)).
+            values(checks_made = Conversations.checks_made + 1)
+        )
+
+        with self.Sessionmaker() as session:
+            session.execute(query)
+            session.commit()
+
+
+    def convert_invocation_conversations(self, data):
+        """ converts invocation dict to correct column names """
+        clean_data = {}
+        try:
+            clean_data['form'] = data.form
+        except KeyError:
+            pass
+        try:
+            clean_data['isChecked'] = True
+        except KeyError:
+            pass
+        try:
+            clean_data['link'] = data.link
+        except KeyError:
+            pass
+        try:
+            clean_data['root_tweet_id'] = data.tweet_id
+        except KeyError:
+            pass
+        try:
+            clean_data['tweeter_id'] = data.user_name
+        except KeyError:
+            pass
+        try:
+            clean_data['user_name'] = data.user_name
+        except KeyError:
+            pass
+        return clean_data
+
+
+    def convert_form_conversations(self, data):
+        """ Converts form dict to correct column names """
+        clean_data = {}
+        clean_data['form'] = 1
+        try:
+            clean_data['root_tweet_id'] = data.tweet_id
+        except KeyError:
+            pass
+        try:
+            clean_data['root_tweet_city'] = data.city
+        except KeyError:
+            pass
+        try:
+            clean_data['root_tweet_state'] = data.state
+        except KeyError:
+            pass
+        try:
+            clean_data['root_tweet_lat'] = data.lat
+        except KeyError:
+            pass
+        try:
+            clean_data['root_tweet_long'] = data.long
+        except KeyError:
+            pass
+        try:
+            clean_data['root_tweet_date'] = data.incident_date
+        except KeyError:
+            pass
+        try:
+            clean_data['root_tweet_force_rank'] = data.force_rank
+        except KeyError:
+            pass
+        try:
+            clean_data['tweeter_id'] = data.user_name
+        except KeyError:
+            pass
+        return clean_data
```

Comment:
Is it possible to not have to do any of this by just having the JSON that we take in to have the same names as the column names?

Reply:
So regarding this one, I'm going to leave this as is because we'd still need to have a function to grab only the keys we want.  I'm unpacking the dict to insert into the DB, and if there's a key in the dict that doesn't correspond to the table model, it throws an error, so even if we change the expected input names, we'd still need this function